apps/autostart-stop.html.md.erb

```diff
@@ -0,0 +1,83 @@
+---
+title: Automatically Stop and Start App Machines
+objective: 
+layout: docs
+nav: firecracker
+order: 65
+---
+
+<%= partial "/docs/partials/v2_transition_banner" %>
+
+You can decide whether you want to automatically stop, or downscale, instances of your app when they're idle and automatically restart them when they're needed. Each instance of your app is running on a Fly machine. Fly Proxy controls automatically starting and automaticaly stopping machines according to settings in your `fly.toml` file, in either the `services` or `http_service` sections:
+
+```toml
+...
+[[services]]
+  autostart = true
+  autostop = true
+...
+```
+
+```toml
+...
+[http_service]
+  internal_port = 8080
+  force_https = true
+  auto_stop_machines = true
+  auto_start_machines = true
+...
+```
+
+## Default and recomended values
+
+New V2 apps are automatically started and automatically stopped by default:
+
+```toml
+auto_stop_machines = true
+auto_start_machines = true
+```
+
+Existing apps are automatically started but not automatically stopped by default:
+
+```toml
+auto_stop_machines = false
+auto_start_machines = true
+```
+
+In most cases, we recommend setting `auto_stop_machines` and `auto_start_machines` to the same value to avoid having instances that either never start or never stop. 
+
+If `auto_start_machines = true` and `auto_stop_machines = false`, the proxy will start your instances but they will never be stopped automatically. If you have a reason to only stop instances manually, then these settings are just fine.
```

Comment:
I think th' is is valid case and we have a chance to link to a form post or even better a doc explaining how to use "self stopping" machines.

Reply:
@dangra Do you have something in mind or is it not written yet? 🙂

Reply:
@senyosimpson gave a good reason at https://community.fly.io/t/automatically-starting-stopping-apps-v2-instances/12342/5?u=dangra